huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_flinkjar_job_test.go

```diff
@@ -62,57 +71,86 @@ func TestAccResourceDliFlinkJarJob_basic(t *testing.T) {
 	})
 }
 
-func testAccFlinkJarJobResource_basic(name, bucketName, region, ak, sk, jarObsPath string) string {
+func testAccFlinkJarJobResource_base(bucketName, name string) string {
 	return fmt.Sprintf(`
 variable "ak" {
   type        = string
   description = "value"
-  default     = "%s"
+  default     = "%[1]s"
 }
 
 variable "sk" {
   type        = string
   description = "value"
-  default     = "%s"
+  default     = "%[2]s"
 }
 
 variable "jarObsPath" {
   type        = string
   description = "value"
-  default     = "%s"
+  default     = "%[3]s"
 }
 
 resource "huaweicloud_obs_bucket" "test" {
-  bucket        = "%s"
+  bucket        = "%[4]s"
   acl           = "private"
   force_destroy = true
 }
 
 resource "huaweicloud_dli_package" "test" {
-  group_name  = "jarPackage"
+  group_name  = "%[5]s"
   type        = "jar"
   object_path = var.jarObsPath
 }
 
 resource "huaweicloud_dli_queue" "test" {
-  name       = "%s"
+  name       = "%[5]s"
   cu_count   = 16
   queue_type = "general"
 
   tags = {
     foo = "bar"
   }
 }
+`, acceptance.HW_ACCESS_KEY, acceptance.HW_SECRET_KEY, acceptance.HW_DLI_FLINK_JAR_OBS_PATH, bucketName, name)
+}
+
+func testAccFlinkJarJobResource_basic(name, bucketName string) string {
+	return fmt.Sprintf(`
+%[1]s
 
 resource "huaweicloud_dli_flinkjar_job" "test" {
-  name            = "%s"
+  name            = "%[2]s"
+  queue_name      = "common_test"
+  entrypoint      = "${huaweicloud_dli_package.test.group_name}/${huaweicloud_dli_package.test.object_name}"
+  entrypoint_args = "--output.path obs://${var.ak}:${var.sk}@obs.%[3]s.myhuaweicloud.com/${huaweicloud_obs_bucket.test.bucket}/output"
+  obs_bucket      = huaweicloud_obs_bucket.test.bucket
+  log_enabled     = true
+
+  tags = {
+	foo = "bar"
```

Comment:
wrong format, please check this and the other places

Reply:
fixed